env_manager/backends/conda_like_interface.py

```diff
@@ -24,14 +36,24 @@ def create_environment(
             command += packages
         if channels:
             command += ["-c"] + channels
-        result = subprocess.check_call(command)
-        print(result)
+        try:
+            result = subprocess.run(
+                command, stderr=subprocess.PIPE, check=True, text=True
+            )
+            print(result.stdout)
+            return (True, result)
+        except Exception as error:
+            return (False, f"{error.returncode}: {error.stderr}")
 
     def delete_environment(self, environment_path):
-        result = subprocess.check_call(
-            [self.executable, "remove", "-p", environment_path, "-y"]
-        )
-        print(result)
+        command = [self.executable, "remove", "-p", environment_path, "--all", "-y"]
+        try:
+            result = subprocess.run(command, capture_output=True, check=True, text=True)
+            print(result.stdout)
+            shutil.rmtree(environment_path)
```

Comment:
Perhaps we should add `ignore_errors=True` here to avoid an error in case we fail to remove the env path?